test/shopify-cli/shopifolk_test.rb

```diff
@@ -0,0 +1,77 @@
+require 'test_helper'
+require 'fileutils'
+
+module ShopifyCli
+  class ShopifolkTest < MiniTest::Test
+    FEATURE_NAME = "shopifolk"
+    def test_correct_features_is_shopifolk
+      ShopifyCli::Feature.disable(FEATURE_NAME)
+      Dir.mktmpdir do |dev_dir|
+        FileUtils.mkdir_p("#{dev_dir}/bin")
+        FileUtils.touch("#{dev_dir}/bin/dev")
```

Comment:
Since we're now stubbing the existing of this, I don't think there's any need to create it?

Reply:
Nope! Will remove!